File: ejercicios/python/ejercicio074.py
def sumador_recursivo(n,m):
   if (n == m):
      return n
   else:
      return n + sumador_recursivo(n+1, m)

if __name__ == "__main__":
   n = int(input("ingrese el primer numero : "))
   m = int(input("ingrese el segundo numero : "))
   if (m >= n):
      print("los numeros entre " + str(n) + " y " + str(m) + " suman " + str(sumador_recursivo(n,m)))
   else:
      print("Error los numeros deben ser ingresados de menor a mayor.")

File: ejercicios/c++/ejercicio074.cpp
#include <iostream>
using namespace std;

int sumador_recursivo(int n, int m){	if(m == n)		return n;	else		return n + sumador_recursivo(n + 1, m);}

int main() {	int numero1, numero2;	cout << "ingrese un numero entero : ";	cin >> numero1;
	cout << "ingrese otro numero entero : ";
	cin >> numero2;	if (numero1 <= numero2)		cout << "los numeros entre " << numero1 << " y " << numero2 << " suman " << sumador_recursivo(numero1, numero2) << endl;	else		cout << "Error los numeros deben ser ingresados de menor a mayor." << endl;	return 0;}
